policy/nomad/source.go

```diff
@@ -213,12 +210,6 @@ func (s *Source) MonitorPolicy(ctx context.Context, req policy.MonitorPolicyReq)
 		case <-blockingQueryCompleteCh:
 		}
 
-		// Return immediately if context is closed.
-		if ctx.Err() != nil {
-			log.Trace("done with policy monitoring")
-			return
-		}
```

Comment:
is this no longer needed?

Reply:
It didnt make a lot of sense to begin with